desc/objectives/_free_boundary.py

```diff
@@ -265,35 +266,44 @@ def print_value(self, *args, **kwargs):
 
         abserr = jnp.all(self.target == 0)
 
-        def _print(fmt, fmax, fmin, fmean, norm, units):
+        def _print(fmt, fmax, fmin, fmean, f0max, f0min, f0mean, norm, unit):
```

Comment:
why change `units` to `unit`? seems unnecessary

Reply:
Sorry my english is bad :laughing: